<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpModule</class>
 <widget class="QWidget" name="HelpModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帮助与文档</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="navTabWidget">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="contentsTab">
       <attribute name="title">
        <string>目录</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="contentsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="indexTab">
       <attribute name="title">
        <string>索引</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="indexWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>搜索</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="searchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="searchResultsWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="contentWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="helpBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="contentsButton">
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="indexButton">
       <property name="text">
        <string>索引</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton_2">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>关于</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>HelpModule</receiver>
   <slot>onSearchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contentsButton</sender>
   <signal>clicked()</signal>
   <receiver>HelpModule</receiver>
   <slot>onShowContentsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexButton</sender>
   <signal>clicked()</signal>
   <receiver>HelpModule</receiver>
   <slot>onShowIndexClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton_2</sender>
   <signal>clicked()</signal>
   <receiver>HelpModule</receiver>
   <slot>onShowSearchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>HelpModule</receiver>
   <slot>onAboutButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
